net/nfs-kernel-server/Makefile

```diff
@@ -33,100 +33,124 @@ define Package/nfs-kernel-server/Default
 	SECTION:=net
 	CATEGORY:=Network
 	SUBMENU:=Filesystem
-	DEPENDS:=+libblkid +libuuid +libtirpc +lsqlite3 +libxml2 +libnl
+	DEPENDS:=+libblkid +libuuid +libtirpc +lsqlite3 +libxml2 +libnl +libreadline
```

Comment:
Is libreadline a dependency of v3 too? It wasn't here before v4 effort.

Reply:
good catch, moved